CONTRIBUTING.md

```diff
@@ -184,7 +184,7 @@ CSS class names, custom property names and keyframe names al must start with the
 
 ### Installing and testing packages locally
 
-We have tested this using a verdaccio local registry on the PWA master and development branches.
+We have tested this using a verdaccio local registry on the PWA `main` and `development` branches.
```

Comment:
Technically the `nl-portal-libraries` package still uses a `master` branch instead of a main branch.
However, this whole documentation seems a bit outdated and might need to be reworked (it also refers to the old repo in this section. @bddjong

Reply:
Created issue #533 for this. I'm guessing the readme can also use an update ;)